tests/testthat/test-to_relational_data.R

```diff
@@ -105,9 +105,9 @@ test_that("to_relational_data accepts a function returning a TealDataset as inpu
 test_that("to_relational_data accepts a MultiAssayExperiment as input", {
   utils::data(miniACC, package = "MultiAssayExperiment")
   mae <- miniACC
-  output_dataset <- to_relational_data(mae)
+  output_dataset <- to_relational_data_wrapper(mae)
```

Comment:
@gogonzo why do we need `to_relational_data_wrapper` if it's just a function calling `to_relational_data`
```r
to_relational_data_wrapper <- function(data) {
  to_relational_data(data)
}
```
When it comes to the name - not really a wrapper, but rather an identity function. Wrapper would do some slight preparations or modifications.

Reply:
This will be removed soon. Let's skip the discussion about this

Reply:
coolio